src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -30,6 +30,9 @@ public interface IBuildFileNames
 
     public class FileNameBuilder : IBuildFileNames
     {
+        private static readonly string[] ValidHdrTransferFunctions = {"PQ", "HLG"};
```

Comment:
Architecturally, should FileNameBuilder know about specific Transfer Functions? Take a look at MediaInfoFormatter.

Reply:
Yes, good point, I have to admit I was just copying what was in Radarr.